.github/actions/dbp-charts/verify-helm/action.yml

```diff
@@ -50,12 +50,16 @@ inputs:
     description: "version of the kubectl CLI"
     required: false
     default: 'v1.23.6'
-
+  skip_checkout:
+    description: "Do not checkout the branch again"
+    required: false
```

Comment:
add a default

Reply:
I did not put one as the input var is only used in the `if` statement so result would be the same

Reply:
I would never trusted github to implicitly default to false